.gitignore

```diff
@@ -2,4 +2,4 @@
 node_modules/*
 rethinkdb_data/*
 npm-debug.log
-public/*
+dist/*
```

Comment:
💡 Btw, while you are renaming it you might as well add `.gitkeep` inside it to prevent git from deleting empty folder.